### PR TITLE
Add Geode driver

### PR DESCRIPTION
**Pull request by sbawaska**
This is a retry of #388.

### CODE REVIEW

**Comment by busbey** [comment]
please update the most recent set of commits so their commit message starts with the component impacted, e.g. "[geode]".

File: geode/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!--
+Copyright (c) 2014 - 2015 YCSB contributors. All rights reserved.
```
Comment (busbey) [inline geode/README.md]: some of these edits are 2016, so the year range should include it.

File: geode/pom.xml [hunk 2]
```diff
@@ -47,10 +47,4 @@ LICENSE file.
     </dependency>
   </dependencies>
   
-  <repositories>
-    <repository>
-      <id>gemstone</id>
-      <url>http://dist.gemstone.com.s3.amazonaws.com/maven/release</url>
```
Comment (busbey) [inline geode/pom.xml]: :clap:

File: geode/src/main/java/com/yahoo/ycsb/db/GeodeClient.java [hunk 3]
```diff
@@ -64,23 +64,23 @@
  * @author Swapnil Bawaskar (sbawaska at vmware)
```
Comment (busbey) [inline geode/src/main/java/com/yahoo/ycsb/db/GeodeClient.java]: while you're here, would you mind clearing out @author tags? we rely on git commit metadata for authorship attribution.

**Comment by busbey** [comment]
I'd say this looks good to go, once the couple of nits above are addressed.

Thanks for sticking with getting this in, and kudos to the Geode community on their first release (and some impressive API compatibility)

Note to whomever ends up managing the next release: this should be labeled as a breaking change.

**Comment by sbawaska** [comment]
Thanks for taking a look Sean! I have incorporated your feedback.

**Pull request merged**